test/e2e/core.go

```diff
@@ -50,11 +50,16 @@ func CoreDump(dir string) {
 		return
 	}
 
-	cmds := []command{
-		{"cat /var/log/kubelet.log", "kubelet"},
-		{"cat /var/log/kube-proxy.log", "kube-proxy"},
-		{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+	cmds := []command{{"cat /var/log/kube-proxy.log", "kube-proxy"}}
+	if isUsingSystemdKubelet(provider, hosts...) {
+		cmds = append(cmds, command{"sudo journalctl --output=cat -u kubelet.service", "kubelet"})
```

Comment:
we could factor out "kubelet.service" into a constant, sure. how likely is this to change, though?